certbot/lock.py

```diff
@@ -80,7 +80,9 @@ def _try_lock(self, fd):
                 logger.debug(
                     "A lock on %s is held by another process.", self._path)
                 raise errors.LockError(
-                    "Another instance of Certbot is already running.")
+                    "Another instance of Certbot is already running. If for "
+                    "some reason you really want to run two Certbots simultaneously, "
+                    "set --certbot-dir to a different value.")
```

Comment:
This seems distinct enough from adding `--certbot-dir` that it probably belongs in a separate PR.

It also seems like @bmw's comment from #4655, https://github.com/certbot/certbot/pull/4655#pullrequestreview-37966820, still applies:

> Additionally, I think this is the wrong place for this error message. If we fail to acquire a lock for the Apache or Nginx plugins, this exception about `--config-dir` and friends gets raised. Instead, if you'd like to see an error message like this, I think it should be added to `certbot.util.set_up_core_dir` by having us catch the `LockError` and raise a new error message. We already do this there in the event of a permission problem so adding another case for a `LockError` fits nicely in my opinion.